.surge.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+set -x
+
+if [ "$TRAVIS_EVENT_TYPE" = "push" -a "$TRAVIS_BRANCH" = "master" ]; then
+  docker run -it -e TRAVIS=1 -e TRAVIS_JOB_ID="$TRAVIS_JOB_ID" \
+    -e SURGE_LOGIN="$SURGE_LOGIN" -e SURGE_TOKEN="$SURGE_TOKEN" \
```

Comment:
ok, it will mainly fail, because you do not have permission to that surge page. otherwise simple `surge` will work in interactive mode.